docs/UserGuide.adoc

```diff
@@ -630,13 +637,24 @@ The `undo` command fails as there are no undoable commands executed previously
 `undo` (reverses the `clear` command) +
 `undo` (reverses the `delete 1` command) +
 
+* `delete 1` +
+`undo` (reverses the `delete 1` command) +
+`clear`(`delete 1` commmand will no longer be undoable as it is already undone) +
+`undo` (reverses the `clear` command) +
+`undo` (no more commands to undo) +
+
 [NOTE]
 ====
 Undoable commands: those commands that modify CHRS content.
 For commands similar functions to +
 (list*, find*, filter*, select*, calculate*)
 the command will not be able to undo or redo).
 ====
+=== Redoing the previously undone command : `redo`
+
+Reverses the most recent `undo` command.
+
+Format: `redo`
```

Comment:
Redo command has been covered in the user guide already, this seems to be a repeated coverage. Do check and recommit

Reply:
fixed, yup i dunno y it duplicated twice